css/about.css

```diff
@@ -51,43 +42,16 @@
   font-size: 1.2rem;
   font-weight: 700;
   padding: 1rem 0.1rem;
-  box-shadow: 20px 20px 20px #e3e6e9;
```

Comment:
I will prefer that you don't change `box-shadow`

Reply:
![Screenshot from 2021-06-21 18-22-32](https://user-images.githubusercontent.com/79043763/122764990-ab541b80-d2bd-11eb-9464-9b5f0b2d02e8.png)
Keeping the box shadow like that makes the numbers part look a little weird.

Reply:
Without the above shadow it looks like this
![Screenshot from 2021-06-21 18-32-31](https://user-images.githubusercontent.com/79043763/122766364-0f2b1400-d2bf-11eb-81d4-dd1925c2de2b.png)